README.md

```diff
@@ -16,6 +16,10 @@ Percy-node is an installable [npm package](https://www.npmjs.com/package/percy-n
 npm install percy-node --save-dev
 ```
 
+## Feature
+
+Percy-node provides the feature to return Percy build results. To enable that feature, please reach out to the Percy team to get a token with read access.
```

Comment:
"Percy-node provides an optional feature..."

Reply:
add additional notes to this documentation showing how to enable the feature (passing true to the build function)